hack/jenkins/e2e.sh

```diff
@@ -217,6 +217,9 @@ if [[ "${E2E_UP,,}" == "true" ]]; then
         # other.
         export KUBE_SKIP_UPDATE=y
         sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update -q" || true
+        sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update preview -q" || true
+        sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update alpha -q" || true
```

Comment:
The code we're replacing might update either:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/gke/util.sh#L101

It seemed safe to update all given we're skipping that code. Is there any reason not to?